substrate/frame/broker/src/tests.rs

```diff
@@ -449,6 +449,18 @@ fn renewals_affect_price() {
 	});
 }
 
+// In the following payouts tests, assertions of the pot balances currently account for the fact
+// that the pot is topped up twice: first, when the credits are purchased, the value is transferred
+// from the buyer's account to the pot (and currently that doesn't really result in any credits
+// being acquired on the relay chain as the credit system is not implemented yet), and second,
+// when the revenue is teleported from the relay chain to the parachain, thus minting the revenue
+// value into the pot. Thus, current pot assertions are not correct and should be fixed back when
+// the credit system is implemented.
+//
+// The other way to handle that wold be burning the balance transferred from the buyer's account to
+// the pot in `do_purchase_credit()`, but, as it would affect the code that will go into
+// production, it's more error prone, so handling incorrect pot balance is the lesser evil.
```

Comment:
Or we just comment out the `do_purchase_credit` call?

Reply:
In principle, yes, but that would require some severe changes to the mock implementation as well, as `TestCoretimeProvider::spend_instantaneous()` behaves like the credit system is the thing. But it would definitely be a cleaner solution, I'll look into how easy it is to implement.

Reply:
Yes, you were right. The changes were not that scary, and the tests are much cleaner now. At least there's no need to guess where the numbers come from.